python/snet_deframer.py

```diff
@@ -11,9 +11,10 @@
 from gnuradio import gr
 import numpy as np
 import pmt
+from construct import *
 
 from .bch15 import decode_bch15
-from .telemetry.snet import LTUFrameHeader
+from .telemetry.snet import LTUFrameHeader, PDUFrameHeader
```

Comment:
If we don't use the `PDUFrameHeader` below, then we do not need to change the import.